parser/src/util.ts

```diff
@@ -0,0 +1,32 @@
+import { spawn } from "child_process"
+import { promises as fs } from "fs"
+import { resolve, sep } from "path"
+import { GitCommitObject } from "./model.js"
+
+export function runProcess(command: string, args: string[]) {
+  return new Promise((resolve, reject) => {
+    const prcs = spawn(command, args)
+    prcs.stdout.once("data", (data) => resolve(data.toString()))
+    prcs.stderr.once("data", (data) => reject(data.toString()))
+  })
+}
+
+export async function writeCommitObjectToFile(
```

Comment:
Should this be renamed to something like "writeRepoToFile"?
It start from the head commit, but i think the name is a bit misleading, as the function leads to everything being printed (including trees and such). Or have i misunderstood the function?